charts/seed-bootstrap/templates/fluentd-es/fluentd-es-ds.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd-es
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: fluentd-es
+    role: logging
+---
+kind: ClusterRole
+apiVersion: {{ include "rbacversion" . }}
+metadata:
+  name: fluentd-es
+  labels:
+    app: fluentd-es
+    role: logging
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "namespaces"
+  - "pods"
+  verbs:
+  - "get"
+  - "watch"
+  - "list"
+---
+kind: ClusterRoleBinding
+apiVersion: {{ include "rbacversion" . }}
+metadata:
+  name: fluentd-es
+  labels:
+    app: fluentd-es
+    role: logging
+subjects:
+- kind: ServiceAccount
+  name: fluentd-es
+  namespace: {{ .Release.Namespace }}
+  apiGroup: ""
+roleRef:
+  kind: ClusterRole
+  name: fluentd-es
+  apiGroup: ""
+---
+apiVersion: {{ include "daemonsetversion" . }}
+kind: DaemonSet
+metadata:
+  name: fluentd-es
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: fluentd-es
+    role: logging
+spec:
```

Comment:
Add rolling update strategy to the DaemonSet. It's `OnDelete` by default.

```yaml
  updateStrategy:
    type: RollingUpdate
```

Reply:
Added.